src/client/eppo-client.ts

```diff
@@ -228,7 +237,7 @@ export default class EppoClient {
     );
     return {
       value: eppoValue.stringValue ?? defaultValue,
-      ...flagEvaluationDetails,
+      evaluationDetails: flagEvaluationDetails,
```

Comment:
I wonder if for this, and bandits, we just return the `flagEvaluationDetails` rather than require another layer of dereferencing. It should have the assigned variation (and action for bandits) there.

Reply:
chatted about this synchronously - we decided to keep the key